app/src/org/commcare/views/DotsHomeView.java

```diff
@@ -84,7 +86,7 @@ private void refresh() {
             table.addView(row);
         }
 
-        Button done = new Button(this.getContext());
+        Button done = new AppCompatButton(this.getContext());
         done.setId(666);
         done.setText("Finished");
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/5a959df2d1681e0a640ebb72d4513eaecefa585e/app/src/org/commcare/views/DotsHomeView.java#L91"></span>String literal in <code>setText</code> can not be translated. Use Android resources instead.</td>
    </tr>
  </tbody>
</table>